migrations/20231210123554_init.sql

```diff
@@ -0,0 +1,323 @@
+    is_editable  BOOLEAN NOT NULL,
+    manager_note TEXT,
+    -------------------------------------------------------
+    created_at   TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at    TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at   TIMESTAMP,
+    --------------------------------------------------------
+    user_id      UUID NOT NULL,
+    company_id   UUID NOT NULL,
+    event_id     UUID NOT NULL,
+    --------------------------------------------------------
+    FOREIGN KEY  (user_id) REFERENCES user_record (id),
+    FOREIGN KEY  (company_id) REFERENCES company (id),
+    FOREIGN KEY  (event_id) REFERENCES event (id),
+    --------------------------------------------------------
+    CONSTRAINT check_timesheet_start_date_lte_end_date
+        CHECK (start_date >= end_date),
+    CONSTRAINT check_timesheet_created_at_lte_edited_at
+        CHECK (edited_at >= created_at)
+);
+
+
+CREATE TABLE work_day
+(   
+    id           UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    --------------------------------------------------------
+    date         DATE NOT NULL,
+    total_hours  hours_per_day_float NOT NULL,
+    comment      TEXT,
+    is_editable  BOOLEAN NOT NULL,
+    --------------------------------------------------------
+    created_at   TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at    TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at   TIMESTAMP,
+    --------------------------------------------------------
+    timesheet_id UUID NOT NULL,
+    --------------------------------------------------------
+    FOREIGN KEY  (timesheet_id) REFERENCES timesheet (id),
+    --------------------------------------------------------
+    CONSTRAINT check_work_day_created_at_lte_edited_at
+        CHECK (edited_at >= created_at)
+);
+
+
+-- WORKFLOW ENTITIES
+
+CREATE TABLE event_staff
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    role        event_role NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    staff_id     UUID NOT NULL,
+    event_id    UUID NOT NULL,
+    -------------------------------------------------------
+    FOREIGN KEY (staff_id) REFERENCES user_record (id),
+    FOREIGN KEY (event_id) REFERENCES event (id),
+    -------------------------------------------------------
+    CONSTRAINT check_event_stuff_created_at_lte_edited_at
+        CHECK (edited_at >= created_at)
+);
+
+
+CREATE TABLE task
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    title           VARCHAR(255) NOT NULL,
+    description     TEXT,
+    finished_at     DATE,
+    priority        task_priority NOT NULL,
+    accepts_staff   acceptance_status NOT NULL,
```

Comment:
This is a boolean value. Either we accept staff, or we don't.
What you are mistaking this with is the assigned_staff table in the ERD.